tests/tests/util/runner.rs

```diff
@@ -13,6 +18,31 @@ use std::rc::Rc;
 use std::sync::{Arc, Mutex};
 use std::time::Duration;
 
+pub struct TestAudioBackend {
+    mixer: AudioMixer,
+}
+
+impl TestAudioBackend {
+    pub fn new() -> Self {
+        let mixer = AudioMixer::new(2, 44100);
+        let mixer_proxy = mixer.proxy();
+
+        std::thread::spawn(move || loop {
+            let mut buffer = [0f32; 2 * 1024];
+            mixer_proxy.mix::<f32>(&mut buffer);
+            std::thread::sleep(std::time::Duration::from_millis(1000 * 1024 / 44100));
```

Comment:
Just for my own sanity, can you extract 44100 into a const, I'm guessing that's `SAMPLE_RATE`?
What's 1024?

Reply:
Already did this yesterday, just didn't commit/push then... :)